.eslintrc

```diff
@@ -1,6 +1,6 @@
 {
   "parser": "babel-eslint",
-  "plugins": ["react", "mozilla", "flowtype", "babel", "prettier"],
+  "plugins": ["react", "mozilla", "flowtype", "babel", "prettier", "import"],
```

Comment:
Here are the plugin docs: https://github.com/benmosher/eslint-plugin-import

We are using this in perf.html, and it's been really helpful.